src/main/java/io/reactivesw/category/infrastructure/validator/CategoryNameValidator.java

```diff
@@ -30,23 +32,30 @@ private CategoryNameValidator() {
 
   /**
    * Validate null.
-   *
    * @param draft the draft
    */
   public static void validateNull(CategoryDraft draft) {
 
     if (draft.getName() == null
         || draft.getName().getLocalized() == null
         || draft.getName().getLocalized().isEmpty()) {
-      LOG.debug("CategoryView name can not be null");
-      throw new ParametersException("CategoryView name can not be null");
+      LOG.debug("CategoryView name can not be null.");
+      throw new ParametersException("CategoryView name can not be null.");
     }
+    Predicate<Map.Entry<String, String>>
+        keyPredicate =
+            (entry) -> StringUtils.isEmpty(entry.getValue()) || StringUtils.isEmpty(entry.getKey());
+    if (draft.getName().getLocalized().entrySet().stream()
+        .anyMatch(entry -> keyPredicate.test(entry))) {
+      LOG.debug("CategoryView name can not be empty.");
+      throw new ParametersException("CategoryView name can not be empty.");
+    }
+
```

Comment:
why use anyMatch(entry -> keyPredicate.test(entry))? I think anyMatch(keyPredicate) is ok.